tests/tests.cpp

```diff
@@ -57,13 +58,13 @@ int main()
             case 0x0602: std::cout << "8"; break;
             case 0x0603: std::cout << "8.1"; break;
             case 0x0A00: std::cout << "10"; break;
-            default: std::cout << (WINVER > 0x0A00) ? "10+" : "NT or earlier";
+            default: std::cout << "10+";
         }
         std::cout << "\n";
     }
 
     {
-        LOG("Testing serialization and hashing for thread::id...");
+        LOG("%s","Testing serialization and hashing for thread::id...");

```

Comment:
Please don't change these, the artificial %s is not necessary

Reply:
It spawns a compiler warning. ISO C++ technically requires the `...` of a variadic macro to contain at least one argument. Because this is part of tests.cpp, rather than the library itself, it could be reverted safely, if desired.

Reply:
Okay, if yoy get a warning then let's keep the %s.